docs/system-overview.md

```diff
@@ -0,0 +1,77 @@
+_This document is intended to read top-to-bottom. Do yourself a favour and read the whole thing without skipping ;)_
+
+# Formula Student Driverless Simulation: System overview
+
+FSDS is built around Unreal Engine 4 (the game engine) and the AirSim plugin. 
+The game engine does all the graphical rendering, collision simulation and car movement simulation. 
+A separate component - the simulator - will handle all control of the simulation, provide external interfaces and store what is happening.
+
+! We will also provide a video feed of the simulation for live streaming purposes.
+  Since it is still unclear how this will work this is not included in this description.
+
+![System overview](images/system-overview.png)
+
+## Autonomous Systems.
+
+Every autonomous system runs on its own separated environment. 
+Idealy this would be separate virtual machines but also Docker containers could be used.
+The autonomous systems are expected to continuously run a ROS master.
+When the simulator is ready to do a drive for the given autonomous system, the simulator will launch a ROS node connected to this autonomous system's ROS master.
+This ROS node will exist outside of the autonomous system's environment, instead it will run on the simulator computer.
+The simulator ROS node will publish sensor data and listen for car setpoints on a set of topics defined [here](airsim-ros-wrapper.md).
+When the simulation is finished, the ROS node will disconnect from the autonomous system's ROS master and sensor data is no longer streaming.
+
+During the competition the teams will not be allowed to access their autonomous systems.
+All remote access to the environments will be cut off completely.
+So every autonomous system must be able to run multiple missions without human interferance.
+To let the simulation know what is expected from it, the simulator will send mission control messages a few seconds before the event start.
+These messages contain information about the mission (trackdrive, autocross, etc).
+When the autonomous system receives a mission message it can expect to receive sensor data shortly after.
+Since the laptimer for all events start whenever the car crosses the start-line, the autonomous system's can take all the time they need to launch their relevant algorithms for the mission (within reasonable time).
```

Comment:
```suggestion
Since the laptimer for all events starts whenever the car crosses the start-line, the autonomous systems can take all the time they need to launch their relevant algorithms for the mission (within reasonable time).
```